config/settings/development.yml

```diff
@@ -1,2 +1,6 @@
-authentication:
-  secret: secret
+manage_api:
```

Comment:
Keep this file as it was, its for local development only

Reply:
done